griptape/loaders/email_loader.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+from attr import astuple, define, field
+from griptape import utils
+from griptape.artifacts import ErrorArtifact, ListArtifact, TextArtifact
+from griptape.loaders import BaseLoader
+from typing import Optional
+import imaplib
+import logging
+import mailparser
+
+
+@define(frozen=True)
+class EmailQuery:
```

Comment:
Can we move this inside `EmailLoader`? Griptape convention is one class per file.